app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsView.kt

```diff
@@ -66,13 +75,19 @@ class RecentPatientsView(
     )
   }
 
+  init {
```

Comment:
Why did we remove the layout binding from the onFinishInflate and adding it in the init method here?

Reply:
In this case, we are inflating the layout. If you look at some of the screens where we are setting the view binding you will see the class is used as a parent in the layout file. So that's why we `bind` it in the `onFinishInflate` method, or else it wouldn't have access to the layout fields. But since here we are inflating the layout file, we can do it in the `init` method.